### PR TITLE
feat : added filter

### PR DESCRIPTION
**Pull request by Omkar-Chaskar**
added filter with sort by date and filter by priority and tags.

### CODE REVIEW

File: src/style/addnote.css [hunk 1]
```diff
@@ -32,4 +31,86 @@
...
+  justify-content: center;
+}
+
+.tags-input {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  flex-wrap: wrap;
+  min-height: 48px;
+  width: 480px;
```
Comment (parulgupta2001) [inline src/style/addnote.css:49]: use rem

File: src/style/addnote.css [hunk 2]
```diff
@@ -32,4 +31,86 @@
...
+#tags {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0;
+  margin: 8px 0 0 0;
+}
+
+.tag {
+  width: auto;
+  height: 32px;
```
Comment (parulgupta2001) [inline src/style/addnote.css:83]: use rem

File: src/style/addnote.css [hunk 3]
```diff
@@ -32,4 +31,86 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #fff;
```
Comment (parulgupta2001) [inline src/style/addnote.css:87]: use var

**Review by parulgupta2001 — APPROVED** [review]
some changes are suggested, otherwise everything looks good

**Review by shristabaruah — APPROVED** [review]

**Review by simran61 — APPROVED** [review]
Looks good
Functionality working as expected

**Pull request merged**